src/Styles/Header.scss

```diff
@@ -0,0 +1,112 @@
+ul {
+  navelement {
+    font-weight: 700;
+    font-family: Open Sans,sans-serif;
+  }
+}
+.header {
+  position: relative;
+  width: 100%;
+  background-color: white;
+}
+.header {
+  .navigation {
+    padding: 8px 0;
+    position: relative;
+    border-bottom: 1px solid #ebf2f2;
+  }
+}
+.container {
+  padding: 0 20px;
+  max-width: 1140px;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.container > ul {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  list-style-type: none;
+  width: 705px;
+  padding: 0;
+}
+.container {
+  div {
+    img {
+      height: 48px;
+      max-width: 250px;
+    }
+  }
+}
+.navElementContainer {
+  position: relative;
+  padding-top: 15px;
+  cursor: pointer;
+}
+.navElementContainer {
+  li {
+   div {
+     span {
+       transition-duration: 0.3s;
+       font-weight: 700;
+     }
+   }
+  }
+}
+@media screen and (max-width: 1100px)  {
+  .container {
+    div {
+      img {
+        height: 30px;
+      }
+    }
+  }
+}
+.arrow {
+  -webkit-transform: rotate(45deg) translateY(-6px);
+  border: solid #191e1e;
+  border-width: 0 1px 1px 0;
+  display: inline-block;
+  padding: 3px;
+  margin-left: 5px;
+}
+.burger-icon {
```

Comment:
This is the proper CSS class name style. Not the camelCased one. Thus, you have a style mismatch in one-man project.